dominion-pax-exam-driver/src/main/java/org/codice/dominion/pax/exam/options/extensions/AbstractFileExtension.java

```diff
@@ -23,38 +23,44 @@
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.io.FilenameUtils;
 import org.codice.dominion.options.OptionException;
+import org.codice.dominion.pax.exam.options.KarafDistributionConfigurationFileRetractOption;
 import org.codice.dominion.pax.exam.options.PaxExamOption.Extension;
 import org.codice.dominion.resources.ResourceLoader;
 import org.ops4j.pax.exam.Option;
+import org.ops4j.pax.exam.karaf.options.KarafDistributionConfigurationFileReplacementOption;
 import org.ops4j.pax.exam.karaf.options.KarafDistributionOption;
 
 /** Provides some common utility methods for dealing with file-based options. */
 public abstract class AbstractFileExtension<A extends Annotation> implements Extension<A> {
-  protected Option[] fileOptions(String source, String target) {
-    return fileOptions(new File(FilenameUtils.separatorsToSystem(source)), target);
+  protected KarafDistributionConfigurationFileReplacementOption fileOption(
```

Comment:
ℹ️ Only changed the methods to return the single option such that it could be reusable